### PR TITLE
Feat : added P1 - Modal 

### PR DESCRIPTION
**Pull request by TusharJade**
### Added Modal component
Screenshot (46)
Screenshot (45)

### CODE REVIEW

File: Components/Modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            <li class="nav-link"><a class="nav-anchor"
+                    href="/Components/Text-Utilities/text-utility.html">Text-Utilities</a></li>
+            <li class="nav-link"><a class="nav-anchor active" href="/Components/Modal/modal.html">Modal</a></li>
+        </ul>
+    </nav>
+    <section class="main-container">
+
+        <h1 class="main-heading">Modal</h1>
+        <div class="separator"></div>
+        <h3 class="sub-heading">Simpel Modal</h3>
```
Comment (krgarima) [inline Components/Modal/modal.html]: Spelling mistake

**Review by krgarima — APPROVED** [review]
Fix the spelling mistake and add cursor pointer to the buttons. Rest is good.

File: Components/Modal/modal.html (hunk 1 above)
Comment (TusharJade) [reply Components/Modal/modal.html]: will fix it thanks 🙌

File: Components/Modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+.modal-btn-save {
+    border: none;
+    background-color: var(--light-blue);
+    color: var(--text-white-color);
+    font-size: 0.9rem;
+    padding: 0.4rem 1rem;
+    border-radius: 0.3rem;
+}
+
+.modal-box-delet{
```
Comment (Avin008) [inline Components/Modal/modal.css]: fix delete spelling

File: Components/Modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+
+        <iframe src="https://www.thiscodeworks.com/embed/623f0d589f69df00152301c3" class="iframe"
+            frameborder="0"></iframe>
+
+        <h3 class="sub-heading">Logout modal</h3>
+        <p class="des-para">You can show this model when your wants to logout from your app.</p>
+
+        <div class="modal-outerbox">
+            <div class="modal-box-delet">
```
Comment (Avin008) [inline Components/Modal/modal.html:67]: here too... e is missing in delete

**Review by Avin008 — APPROVED** [review]
well structured code, good to merge

File: Components/Modal/modal.css (hunk 2 above)
Comment (TusharJade) [reply Components/Modal/modal.css]: yup will fix it now thanks 🙌

**Pull request merged**